Mathlib/FieldTheory/KummerExtension.lean

```diff
@@ -272,6 +272,11 @@ theorem Polynomial.separable_X_pow_sub_C_of_irreducible : (X ^ n - C a).Separabl
     AdjoinRoot.algebraMap_eq,
     X_pow_sub_C_eq_prod (hζ.map_of_injective (algebraMap K _).injective) hn
     (root_X_pow_sub_C_pow n a), separable_prod_X_sub_C_iff']
+  #adaptation_note
+  /--
+  After https://github.com/leanprover/lean4/pull/5376 we need to provide this helper instance.
+  -/
+  have : MonoidHomClass (K →+* K[n√a]) K K[n√a] := inferInstance
```

Comment:
I can't figure out this one yet. Kummer extension notation (or maybe just `AdjoinRoot` itself) loses the coefficient ring in the search during `IsPrimitive.map_of_injective` resulting in a search for rings structures on a metavariable.

Reply:
The notation itself is not great because `K` is not `K` from the context.